ts/components/screens/ScreenContent.tsx

```diff
@@ -0,0 +1,18 @@
+import { Content } from "native-base";
+import React, { ComponentPropsWithRef } from "react";
+
+type Props = ComponentPropsWithRef<typeof Content>;
+
+/**
+ * Wrapper for the native-base Content component that
+ * properly sets default content padding.
+ */
+class ScreenContent extends React.Component<Props> {
```

Comment:
why not naming this `Content` to avoid having to change all the TSX code, we can just change the imports...

on a side note, can't this be fixed by patching NB?